### ISSUE TITLE
Pr.1: La función  vieneDespues analiza una explosion combinatoria de casos

### ISSUE THREAD

**Issue by walterggomez** [Gonzalosanto/EstrD2023s1#3]
No es adecuado escribir funciones que tengan una cantidad cuadrática de casos. Deberías considerar definir subtareas adecuadas cuando te pasa esto…

**Comment by Gonzalosanto** [comment]
Claro, quise pensar en subtareas adecuadas pero no me salian y menos pensarlo de manera que fueran solo 9 casos entre subtareas y la funcion vieneDespues

**Comment by walterggomez** [comment]
podrías pensar en recodificar los días de la semana por ejemplo
o la otra es como hiciste poner todos los PM que hay y luego buscar algún patrón que se repita
